### ISSUE TITLE
Causes extraneous output in dbext

### ISSUE THREAD

**Issue by collinpeters** [airblade/vim-gitgutter#157]
When I have vim-gitgutter enabled, it seems to change something with the output from dbext [1]. dbext is a plugin which allows execution of SQL from Vim. In my case it is using psql for Postgres to execute the query and return the results. These psql commands when manually run, and when run from Vim without gitgutter, return 0 normally. However, when I have gitgutter enabled the return code seems to be 128. This causes dbext to treat it as an error response and print out some debug text including the query, etc..

I asked the other as I originally thought it was their issue and he showed the following code

```
   " If there was an error, show the command just executed
        " for debugging purposes
        if (v:shell_error && l:db_type !~ '\<DBI\>\|\<ODBC\>') ||
                    \ (dbi_result == -1 && l:db_type =~ '\<DBI\>\|\<ODBC\>') 
            let output = "To change connection parameters:\n" .
                        \ ":DBPromptForBufferParameters\n" .
                        \ "Or\n" .
                        \ ":DBSetOption user\|passwd\|dsnname\|srvname\|dbname\|host\|port\|...=<value>\n" .
                        \ ":DBSetOption user=tiger:passwd=scott\n" .
                        \ "Last command(rc=".v:shell_error."):\n" .
                        \ a:cmd . "\n" .
                        \ "Last SQL:\n" .
                        \ a:sql . "\n" 
```

So basically if vshell_error is non-zero, output the error. If I do GitGutterDisable, run the query, I see no error output. If I do GitGutterEnable, run the query, I see the error output. Any ideas on this? I'm loving GitGutter

[1] http://www.vim.org/scripts/script.php?script_id=356
